src/libprojectM/Renderer/Renderer.cpp

```diff
@@ -763,8 +899,7 @@ void Renderer::CompositeOutput(const Pipeline& pipeline, const PipelineContext&
 
 	glBlendFunc(GL_SRC_ALPHA, GL_ONE_MINUS_SRC_ALPHA);
 
-	for (std::vector<RenderItem*>::const_iterator pos = pipeline.compositeDrawables.begin(); pos
-	     != pipeline.compositeDrawables.end(); ++pos)
+	for (std::vector<RenderItem*>::const_iterator pos = pipeline.compositeDrawables.begin(); pos!= pipeline.compositeDrawables.end(); ++pos)
```

Comment:
`for (auto drawable : pipeline.compositeDrawables)`

Reply:
done